community/xwallpaper/sources

```diff
@@ -0,0 +1 @@
+https://github.com/stoeckmann/xwallpaper/archive/v0.6.2.tar.gz
```

Comment:
https://github.com/stoeckmann/xwallpaper/releases/download/v0.6.2/xwallpaper-0.6.2.tar.xz

Then you can drop `auto*` I believe.